forgerock-auth/src/main/java/org/forgerock/android/auth/FROptions.kt

```diff
@@ -147,9 +147,9 @@ data class OAuth(val oauthClientId: String? = "",
  * Oauth builder to build the SDK configuration information specific to oauth
  */
 class OAuthBuilder {
-    var oauthClientId: String? = ""
-    var oauthRedirectUri: String? = ""
-    var oauthScope: String? = ""
+    var oauthClientId: String = ""
```

Comment:
Will the non optional v alus allow the app to be used without setting an OAuth client at all? If they opt to use the FRSession classes, they can run the authentication without running the OAuth flows

Reply:
can we talk about this more, trying to understand this

Reply:
@george-bafaloukas-forgerock  . Both me and @witrisna  tested this flow for both static and dynamic config, we are able to get the SSO Token for FRsession.authenticate. Thanks for bringing this scenario to test